packages/shared/src/styles/components/buttons.css

```diff
@@ -71,14 +77,10 @@
   }
 
   &.iconOnly {
-    width: 2.5rem;
-    padding: 0;
-    border-radius: 0.75rem;
-
-    &.xxsmall {
+    &.none {
       width: 1.25rem;
       padding: 0;
-      border-radius: 0.375rem;
+      border-radius: 0.25rem;
```

Comment:
Do we still need these as you also have them as tailwind classes?
Not quite sure I followed the suggestion here right.

Reply:
We don't need the `.none` styles for now, I'll remove them.